packages/common/src/HospitalUtils.ts

```diff
@@ -9,8 +9,18 @@ export const activeHospitals = [
   Hospital.TEL_HASHOMER,
 ];
 
-export function getAllHospitalOptions(defaultLabel?: string) {
-  return getHospitalOptions(activeHospitals, defaultLabel);
+export const ALL_HOSPITALS_SELECT = "ALL";
+
+export function getAllHospitalOptions(
+  hospitals: Hospital[],
+  allLable: string,
+  defaultLabel?: string
+): SelectOption<Hospital | typeof ALL_HOSPITALS_SELECT | "">[] {
+  let options: SelectOption<Hospital | typeof ALL_HOSPITALS_SELECT | "">[] =
```

Comment:
I see we have `Hospital | typeof ALL_HOSPITALS_SELECT | ""` written in multiple places.

Maybe we should create a type for it:
`type HospitalOptionKey = Hospital | typeof ALL_HOSPITALS_SELECT | ""`

Reply:
done